R/derive_vars_computed.R

```diff
@@ -0,0 +1,194 @@
+#' Adds a Variable Computed from the Analysis Value of one or more Parameters
+#'
+#' Adds a variable computed from the analysis value of one or more parameters.
+#' It is expected that the value of the new variable is defined by an expression
+#' using the analysis values of other parameters. For example Body Mass Index at
+#' Baseline (`BMIBL`) in `ADSL` can be derived from of HEIGHT and WEIGHT
+#' parameters in `ADVS`.
```

Comment:
wouldn't we take the height and weight from `SDTM.VS` rather than ADVS?

Reply:
See my comment below.